friends/src/App.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {Link, Route} from 'react-router-dom'
+import PrivateRoute from './protectedPage/PrivateRoute';
+import Login from './login/LoginPage';
+import ProtectedPage from './protectedPage/ProtectedPage';
+
+function App() {
+  return (
+    <div className="App">
+      <ul>
+        <li>
+          <Link to = '/friendsPage'>Friends Page</Link>
+        </li>
+        <li>
+          <Link to = '/login'>Login</Link>
+        </li>
+      </ul>
+      <Route path = '/login' component = {Login} />
+      <PrivateRoute path = '/friendsPage' component = {ProtectedPage} />
```

Comment:
I was really confused with what ProtectedPage was at first.  I feel that `PrivateRoute` is already doing a great job declaring that it is a protected route.  The component might be better named to what it actually does